modules/uncertainty/R/ensemble.R

```diff
@@ -515,6 +510,8 @@ for (input_tag in names(settings$run$inputs)) {
     for (i in seq_len(ensemble.size)) {
       input_list <- list()
       for (input_tag in names(inputs)) {
+        # if it's the parameter list, skip.
+        if (input_tag == "parameters") next
```

Comment:
Make sure we are not sending parameters for the paths, as the new `inputs_design` does include the `parameter` field.